### PR TITLE
Export cluster specification (gh#367)

### LINKED ISSUE #367: Add info on exporting cluster.yml from already running cluster
`ceph orch ls --export `

### PR DESCRIPTION
**Pull request by tbazant**
Closes #367

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -1902,6 +1902,49 @@ encryption: True
      <xref linkend="deploy-cephadm-day2-orch-status"/>.
     </para>
    </sect3>
+   <sect3 xml:id="cephadm-apply-cluster-specs-">
+    <title>Exporting Specification of a Running Cluster</title>
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1906]:
```suggestion
    <title>Exporting the Specification of a Running Cluster</title>
```

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -1936,21 +1979,21 @@ placement:
...
   - ses-min3
 </screen>
-     <note>
-       <para>
-         If you need add another node, append the host name to the same YAML list.
-         For example:
-       </para>
+    <note>
+     <para>
+      If you need add another node, append the host name to the same YAML list.
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1984]:
```suggestion
      If you need to add another node, append the host name to the same YAML list.
```

File: xml/deploy_cephadm.xml [hunk 3]
```diff
@@ -1902,6 +1902,49 @@ encryption: True
...
+</screen>
+    <tip>
+     <para>
+      You can append the <option>--format</option> option to change the default
+      <literal>yaml</literal> output format. You can choose from
+      <literal>json</literal>, <literal>json-pretty</literal>,
+      <literal>xml</literal>, <literal>xml-pretty</literal>, or
+      <literal>plain</literal>. For example:
+     </para>
+<screen>ceph orch ls --export --format xml</screen>
```
Comment (sebastian-philipp) [inline xml/deploy_cephadm.xml:1945]: 1. for --export, you have the choice between json-pretty, json and yaml.
2. --format xml is in not supported

Comment (tbazant) [inline xml/deploy_cephadm.xml:1945]:
```suggestion
      <literal>json</literal>, <literal>json-pretty</literal>, or
      <literal>plain</literal>. For example:
     </para>
<screen>ceph orch ls --export --format json</screen>
```

File: xml/deploy_cephadm.xml [hunk 4]
```diff
@@ -1902,6 +1902,48 @@ encryption: True
...
+service_type: mgr
+---
+[...]
+</screen>
+    <tip>
+     <para>
+      You can append the <option>--format</option> option to change the default
+      <literal>yaml</literal> output format. You can choose from
+      <literal>json</literal>, <literal>json-pretty</literal>, or
+      <literal>plain</literal>. For example:
```
Comment (tbazant) [inline xml/deploy_cephadm.xml:1942]:
```suggestion
      <literal>json</literal>, <literal>json-pretty</literal>, or
      <literal>yaml</literal>. For example:
```

**Pull request merged**